pkg/pods/pod_manifest.go

```diff
@@ -28,7 +28,7 @@ type LaunchableStanza struct {
 type PodManifest struct {
 	Id                string                      `yaml:"id"` // public for yaml marshaling access. Use ID() instead.
 	LaunchableStanzas map[string]LaunchableStanza `yaml:"launchables"`
-	Config            map[string]interface{}      `yaml:"config"`
+	Config            map[interface{}]interface{} `yaml:"config"`
```

Comment:
In general I don't like using interfaces as keys; if they're uncomparable you get a panic at time of indexing. It doesn't really help that YAML is a weird format and might allow unusual key types (I read the YAML spec front to back once and still don't understand it). The impact is small here and the risk is unclear, so I won't block the PR on this, but personally I would rather enforce string keys everywhere.